### PR TITLE
Set DiskVerification memory usage limit in KubeVirt CR

### PR DESCRIPTION
**Pull request by acardace**
This sets the memory usage limit to a safe value, increasing it
from the default one specified in KubeVirt which can cause occasional
failures when KubeVirt is running on OpenShift.

Refer to https://github.com/kubevirt/kubevirt/pull/6005

Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1979631
Signed-off-by: Antonio Cardace <acardace@redhat.com>

**Reviewer Checklist**

> Reviewers are supposed to review the PR for every aspect below one by one. To check an item means the PR is either "OK" or "Not Applicable" in terms of that item. All items are supposed to be checked before merging a PR. 

- [ ] PR Message
- [ ] Commit Messages
- [ ] How to test
- [ ] Unit Tests
- [ ] Functional Tests
- [ ] User Documentation
- [ ] Developer Documentation
- [ ] Upgrade Scenario
- [ ] Uninstallation Scenario
- [ ] Backward Compatibility
- [ ] Troubleshooting Friendly
  
**Release note**:

```release-note
Set DiskVerification memory usage limit in KubeVirt CR
```

### CODE REVIEW

**Comment by acardace** [comment]
/hold

This PR is will not compile until https://github.com/kubevirt/kubevirt/pull/6005 is merged.

**Review by nunnatsa — CHANGES_REQUESTED** [review]
Please rebase after #1450 is merged

**Comment by acardace** [comment]
> Please rebase after #1450 is merged

Yup, just waiting for that :+1: thank you!

**Comment by nunnatsa** [comment]
> Yup, just waiting for that  thank you!

It's not merged yet :)

**Review by nunnatsa — CHANGES_REQUESTED** [review]
Please follow the inline comments to fix the unit tests, and also, in pkg/controller/operands/kubevirt_test.go, lines ~1118, change from
```golang
existingResource.Spec.Configuration.DeveloperConfiguration = &kubevirtv1.DeveloperConfiguration{
	FeatureGates: fgs,
}
```

to 
```golang
existingResource.Spec.Configuration.DeveloperConfiguration.FeatureGates = fgs
```

File: pkg/controller/operands/kubevirt.go
Comment (acardace) [reply pkg/controller/operands/kubevirt.go:150]: Hi @nunnatsa thanks for the review! I'm afraid this doesn't work as `ParseQuantity` returns `(resource.Quantity, error)` and we cannot deal with a error here.

File: pkg/controller/operands/kubevirt_test.go
Comment (acardace) [reply pkg/controller/operands/kubevirt_test.go]: Changed, thanks!

File: pkg/controller/operands/kubevirt.go
Comment (nunnatsa) [reply pkg/controller/operands/kubevirt.go:150]: But since it's a valid constant string, we can ignore the error (this why the `_` is there).

Comment (acardace) [reply pkg/controller/operands/kubevirt.go:150]: gotcha, will change!

File: pkg/controller/operands/kubevirt.go
Comment (acardace) [reply pkg/controller/operands/kubevirt.go]: Done!

**Pull request merged**
